tests/browser/test/basic_playground_test.js

```diff
@@ -46,3 +51,87 @@ suite('Testing Connecting Blocks', function (done) {
     await browser.deleteSession();
   });
 });
+
+suite('Right Clicking on Blocks', function (done) {
+  // Setting timeout to unlimited as the webdriver takes a longer time to run than most mocha test
+  this.timeout(0);
+
+  // Setup Selenium for all of the tests
+  suiteSetup(async function () {
+    browser = await testSetup(testFileLocations.playground);
+  });
+
+  test('Collapse', async function () {
+    await browser.refresh();
+    const block = await dragNthBlockFromFlyout(browser, 'Loops', 0, 20, 20);
+
+    const blockId = block.id;
+    let isCollapsed = await browser.execute((blockId) => {
+      return Blockly.getMainWorkspace().getBlockById(blockId).isCollapsed();
+    }, blockId);
```

Comment:
I would factor this out into a file-local helper. Same for `isEnabled` and other similar assertions. It keeps the test readable!